dexcom/alert.go

```diff
@@ -304,6 +355,23 @@ func (a *AlertScheduleSettings) Parse(parser structure.ObjectParser) {
 	a.StartTime = parser.String("startTime")
 	a.EndTime = parser.String("endTime")
 	a.DaysOfWeek = parser.StringArray("daysOfWeek")
+	a.Active = parser.Bool("isActive")
+	a.Override = ParseOverrideSetting(parser.WithReferenceObjectParser("override"))
+}
+
+func ParseOverrideSetting(parser structure.ObjectParser) *OverrideSetting {
+	if !parser.Exists() {
+		return nil
+	}
+	datum := &OverrideSetting{}
+	parser.Parse(datum)
```

Comment:
I see this in a couple places and seems a lot of it is based on previous code. Is there a reason not to handle any errors here? Is it because the object will be validated later which takes care of that?

Reply:
yes that is my assumption